### PR TITLE
React capstone project - Metrics web app

### PR DESCRIPTION
**Pull request by Aduda-Boaz**
This React capstone project is about building a mobile web application to check a list of metrics created by making use of React and Redux.
I have made use of the Financial Modelling API to get the stock data of the various companies at the Dow Jones Industrial Average.
The application fetches data according to the requirements.
I have also tested the application and all tests pass.
The design and code also follow all the best practices of JavaScript and HTML/CSS.
No linter errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,62 @@
...
 
-## Available Scripts
+![screenshot](./home.png)
+![screenshot](./company.png)
 
-In the project directory, you can run:
+## Live Demo
 
-### `npm start`
+You can see [the app live here](https://aduda-boaz.github.io/finance-modelling/)
```
Comment (sja-thedude) [inline README.md]: - [x] Kindly make sure your live version works properly. You can also use netlify to deploy your website.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,62 @@
...
+## 🤝 Contributing
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Contributions, issues, and feature requests are welcome!
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Feel free to check the [issues page](https://github.com/Aduda-Boaz/finance-modelling/issues).
 
-### Code Splitting
+Thanks to [Sakwa Nelson](https://www.behance.net/sakwadesignstudio)
```
Comment (sja-thedude) [inline README.md]: - [x] Kindly Replace this with something like `I would like to thank [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).`

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @Aduda-Boaz, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

first review

## Suggested changes
- As you have not added the video for the project, I could not check the following:
- [ ] The recorded video presents the features of the project.
- [ ] The maximum length of the video is 5 minutes.
- [ ] The video contains a clear and objective description of the project in professional English. 
- [ ] The topic and purpose of the project are made clear.
- [ ] The video contains a demo of the project, highlighting all the project features.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
- Your Website have errors:
- [x] Kindly fix your website when I tried running it, it displays an error and I couldn't check the requirements for the project.
Screenshot 2022-05-18 at 3 26 10 PM

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Aduda-Boaz) [reply README.md]: the site works well for me on the localhost
see here
[https://www.loom.com/share/fc6482e7b7fc46fb9ccfce2abb3cf70f](url)

I am working on the changes to the live version.
Thank you for the review

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Aduda-Boaz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
- [x] Kindly fix the error in the localhost as a previous reviewer asked. The problem happen because of the limitation of the API. This is the error I see
image
and when I click on it I get this message
image



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Aduda-Boaz , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes

- [ ] No data is being fetched from the API. I have waited for some time while refreshing the page over and over. No more console error :+1: . Project compiles successively :+1: . Please let's make sure our app is working properly. Check out the screenshot below:


annotely_image (6)


- [Optional] Also, you omitted the License section in your README. Check out [this](https://github.com/microverseinc/readme-template) Microverse README template and include the License section. This will render your README file professional.

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Aduda-Boaz** [comment]
> Hi @Aduda-Boaz ,
> 
> Great work on making the changes clap You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> ## Suggested changes
> * [x]  No data is being fetched from the API. I have waited for some time while refreshing the page over and over. No more console error +1 . Project compiles successively +1 . Please let's make sure our app is working properly. Check out the screenshot below:
> 
> annotely_image (6)
> 
> * [Optional] Also, you omitted the License section in your README. Check out [this](https://github.com/microverseinc/readme-template) Microverse README template and include the License section. This will render your README file professional.
> 
> _Check the comments under the review._
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Hello @FrederickMih I really need help with my API call, I have seen where the problem is. It is because the API only gives details about one company. 
https://financialmodelingprep.com/api/v3/profile/AAPL?......
I need to get the company profiles of other companies but I don't know how. 

I need help please

File: README.md [hunk 3]
```diff
@@ -1,70 +1,70 @@
-# Getting Started with Create React App
+# Stock Company Information
```
Comment (GzimAsani) [inline README.md:1]: - [x] Some of your tests are failing, also please add more tests

File: README.md [hunk 4]
```diff
@@ -1,70 +1,70 @@
...
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> This is a single page web application that retrieves real live data from the Financial Modeling Prep API. Users can get information on their favourite company at Dow Jones Industrial Avarage.
+> Companies are displayed in the Home Page. Information which inclujde the stock index is displayed at the company information section.
 
-## Available Scripts
+![screenshot](./home.png)
+![screenshot](./company.png)
 
-In the project directory, you can run:
+## Live Demo
```
Comment (GzimAsani) [inline README.md:9]: - [x] Your project is not starting on live demo, kindly update it

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @Aduda-Boaz 
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your code organization is good.
 
 ## Required Changes 
- [ ] _Check the comments below :arrow_down_small: _

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Aduda-Boaz) [reply README.md:1]: Hello, all tests pass and I have only three components to test plus the App.js file...

File: README.md (hunk 4 above)
Comment (Aduda-Boaz) [reply README.md:9]: Updated the live demo by deploying to netlify

**Review by bimbolabuari — APPROVED** [review]
Hi @Aduda-Boaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**